demos/sample-javascript/webpack.config.js

```diff
@@ -47,6 +47,11 @@ module.exports = {
     },
   },
   devtool: 'source-map',
+  devServer: {
+    client: {
```

Comment:
Another naive question: where do I find the docs for this? I looked [here](https://webpack.js.org/configuration/dev-server/), but this does not list a 'client' option. 

What does setting `overlay: false` do exactly?

Reply:
> Another naive question: where do I find the docs for this?

Not a naive question at all... as this is JavaScript you must never trust any documentation, because it's all out-of-date by the time it's published. I ended up finding the right config after trial-and-error. 

> What does setting overlay: false do exactly?

It disables a `webpack-dev-server` (that hosts the html/js for the sample JS app) feature, where it puts a screen-wide overlay over your app to display any warnings/errors that occurred during JS bundling. We don't want that for our selenium tests, as the overlay blocks access to the fields/buttons we actually want to fill in/click.

Reply:
> I ended up finding the right config after trial-and-error.

Haha, ok I see. 

> What does setting overlay: false do exactly?
It disables a webpack-dev-server (that hosts the html/js for the sample JS app) feature, where it puts a screen-wide overlay over your app to display any warnings/errors that occurred during JS bundling. We don't want that for our selenium tests, as the overlay blocks access to the fields/buttons we actually want to fill in/click.

I see, thanks for explaining!